democracy_club/templates/base.html

```diff
@@ -70,24 +70,48 @@
 
 <!-- FOOTER -->
 {% block footer_menu %}
-    <div class="ds-cluster-center">
-        <ul>
-            <li><a href="{% url 'home' %}">Home</a></li>
-            <li><a href="{% url 'about' %}">About</a></li>
-            <li><a href="{% url 'projects:home' %}">Our work</a></li>
-            <li><a href="{% url 'hermes_post_list' %}">Blog</a></li>
-            <li><a href="{% url 'contact' %}">Contact Us</a></li>
-            <li><a href="{% url 'jobs' %}">Jobs</a></li>
-            <li><a href="{% url 'privacy' %}">Privacy &amp; Terms</a></li>
-            <li><a href="{% url 'coc' %}">Code of Conduct</a></li>
-            <li><a href="{% url "hermes_post_feed" %}">Blog RSS/Atom feed</a></li>
-            <li><a rel="me" href="https://mastodon.me.uk/@DemocracyClub">Mastodon</a></li>
-            <li><a href="https://twitter.com/democlub">Twitter</a></li>
-            <li><a href="https://github.com/DemocracyClub">GitHub</a></li>
-            <li><a href="https://www.linkedin.com/company/democracy-club-cic">LinkedIn</a></li>
-        </ul>
-    </div>
+    {% block footer_links %}
+        <div class="footer-links">
+            <nav>
+                <h2 class="ds-h6">Made by Democracy Club</h2>
```

Comment:
I'm not sure what's going on with the deployment, but I'm seeing `h6` tags there rather than this code.

Either way, I think the headings can be a bit larger, especially as Peter points out they actually go smaller than the normal font size at lower widths.

Maybe `ds-h5`?

Reply:
@symroe thoughts on mailing list link v CTA?